targets/csharp-unityv2/source/Shared/Models/SharedModels.cs

```diff
@@ -0,0 +1,12 @@
+﻿using UnityEngine;
+using System.Collections;
+
+namespace PlayFab.SharedModels
```

Comment:
Why does this warrant its own file?
Either everything ELSE that's shared should go in here, or this should go back where it was.